### PR TITLE
[eas-cli] Add message about concurrency add-on

### PR DESCRIPTION
**Pull request by brentvatne**
# Why

Let people know that they can add more concurrencies easily

# How

Make the message more clear

# Test Plan

-

### CODE REVIEW

**Comment by brentvatne** [comment]
/changelog-entry chore Let folks know about the new concurrency add-on

**Comment by brentvatne** [comment]
/changelog-entry chore Let folks know about the new concurrency add-on

File: packages/eas-cli/src/build/build.ts [hunk 1]
```diff
@@ -486,8 +486,8 @@ async function handleSingleBuildProgressAsync(
       spinner.succeed('Build finished');
       return { refetch: false };
     case BuildStatus.New:
-      spinner.text = `Build is waiting to enter the queue. Check your concurrency limit at ${link(
-        formatAccountSubscriptionsUrl(accountName)
+      spinner.text = `Build concurrency limit reached for your account, waiting for a concurrency to become available. Add additional concurrencies at ${link(
```
Comment (keith-kurak) [inline packages/eas-cli/src/build/build.ts:489]:
```suggestion
      spinner.text = `Build concurrency limit reached for your account. Build will enter queue once a concurrency becomes available. Add additional concurrencies at ${link(
```
Just an idea... I think the original message communicates slightly better what's happening to the _build_.

**Review by keith-kurak — APPROVED** [review]
Left an idea for a verbiage tweak, but looks good to me!

**Pull request merged**